src/lib/udmp-parser.h

```diff
@@ -1391,10 +1482,11 @@ class UserDumpParser {
       //
       // Update the entry.
       //
-
-      It->second.Data = CurrentData;
-      It->second.DataSize = DataSize;
-      CurrentData += DataSize;
+      {
```

Comment:
Is there a reason for a new scope here?

Reply:
None, debugging artifact left over. Removed.